### ISSUE TITLE
Ele Custom Skin Support going forward

### ISSUE THREAD

**Issue by ultramizer** [dudaster/ele-custom-skin#698]
I think we really need to have a conversation about the future of Ele Custom Skin. My issue is that many bugs are not getting fixed. If this was a free Plugin then one could argue that this is just the way it is. Since i am a paid customer and bought the most expensive plan, i expect some sort of bugfixing when i, as the customer even take my time to open up issues here on the github. With the revenue of the plugin sales there should be some developers invested in this software to make bugfixes happen. I don't expect dudaster to handle everything on his own, but most of the time he doesn't even reply to issues anymore. We have 412 open issues. I am starting to loose complete trust in this plugin to build professional websites and i think we really need to think about the future and have a plan how this plugin is going to be developed in the future so it has a stable base. Am i alone on this?

**Comment by dudaster** [comment]
Most of the issues are Elementor related or other particular cases that the plugin wasn't built for.

Indeed in the past months Elementor made big changes moving lots of the Elementor Pro libraries to Core and this creates issues but I'm trying to be on top of it.

I salute the people who help other find workarounds to the issues that they occur. 

Big issues that I'm working on right now: pagination, and the newest one the custom grid disappearing tab when woocommerce is activated.

Bottom line: I'm on it and trying solving most of the issues.

**Comment by czaczawik** [comment]
> Big issues that I'm working on right now: pagination, and the newest one the custom grid disappearing tab when woocommerce is activated.



I have a problem with a missing tab. When is this bug expected to be fixed?
